packages/utilities/psammead-styles/src/font-families-config.js

```diff
@@ -0,0 +1,17 @@
+const helmet = 'Helmet, Freesans, Helvetica, Arial, sans serif';
+const reithFallback = ', Helvetica, Arial, sans-serif';
+const reithSans = `ReithSans${reithFallback}`;
+const reithSerif = `ReithSerif${reithFallback}`;
+
+const fontFamilies = {
+  latinAndCyrillic: {
```

Comment:
Can you please give examples of this working IRL, latinAndCyrillic isnt [one of the script names exported](https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/scripts.js)